icaruscode/PMT/Trigger/trigger_emulation_icarus.fcl

```diff
@@ -360,6 +360,8 @@ icarus_standard_triggersim: {
       Pattern:               @local::icarus_triggergate_basic.SelectedPattern
       BeamGates:             triggersimgates
       TriggerTimeResolution: "8 ns" # this should probably be 12 or 24 ns
+      EmitEmpty:             true
```

Comment:
I didn't actually check things related to this and the extra info parameter - is there something you would recommend I do to test these @PetrilloAtWork ?

Anyway, since I was added as a reviewer due to CAFs and based on a limited look at things there, I am hitting accept.

Reply:
Not really... `ExtraInfo` causes the production of a `sbn::ExtraTriggerInfo` object that is required, but the timestamps in MC don't have any role. So while it is expected that some timestamp will land into CAF, it should be of no consequence.